src/SamplesApp/UITests.Shared/Windows_System/MemoryManagerTests.xaml

```diff
@@ -0,0 +1,23 @@
+﻿<Page
+    x:Class="UITests.Shared.Windows_System.MemoryManagerTests"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:UITests.Windows_System.MemoryManager"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	xmlns:xamarin="http://uno.ui/xamarin"
+	mc:Ignorable="d xamarin">
+
+	<xamarin:Grid>
```

Comment:
```suggestion
	<Grid>
```
There's likely no need to put this xamarin only. Those APIs are available on Windows, which is why the build fails.

Reply:
Done.